sdk/communication/communication-call-automation/src/callConnection.ts

```diff
@@ -0,0 +1,213 @@
+ */
+export class CallConnection {
+  private readonly callConnectionId: string;
+  private readonly callConnectionImpl: CallConnectionImpl;
+  private readonly callMediaImpl: CallMediaImpl;
+
+  constructor(
+    callConnectionId: string,
+    callConnectionImpl: CallConnectionImpl,
+    callMediaImpl: CallMediaImpl
+  ) {
+    this.callConnectionId = callConnectionId;
+    this.callConnectionImpl = callConnectionImpl;
+    this.callMediaImpl = callMediaImpl;
+  }
+
+  /**
+   * Initializes a new instance of CallMedia.
+   */
+  public getCallMedia(): CallMedia {
+    return new CallMedia(this.callConnectionId, this.callMediaImpl);
+  }
+
+  /**
+   * Get call connection properties
+   */
+  public async getCallConnectionProperties(
+    options: GetCallConnectionPropertiesOptions = {}
+  ): Promise<CallConnectionProperties> {
+    const result = await this.callConnectionImpl.getCall(this.callConnectionId, options);
+    const callConnectionProperties: CallConnectionProperties = {
+      ...result,
+      sourceIdentity: result.sourceIdentity
+        ? communicationIdentifierConverter(result.sourceIdentity)
+        : undefined,
+      targets: result.targets?.map((target) => communicationIdentifierConverter(target)),
+      sourceCallerIdNumber: result.sourceCallerIdNumber
+        ? phoneNumberIdentifierConverter(result.sourceCallerIdNumber)
+        : undefined,
+    };
+    return callConnectionProperties;
+  }
+
+  /**
+   * Hang up the call for itself or terminate the whole call.
+   */
+  public async hangUp(isForEveryOne: boolean, options: HangUpOptions = {}): Promise<void> {
+    if (isForEveryOne) {
+      await this.callConnectionImpl.terminateCall(this.callConnectionId, options);
+    } else {
+      await this.callConnectionImpl.hangupCall(this.callConnectionId, options);
+    }
+    return;
+  }
+
+  /**
+   * Get a participant from the call
+   */
+  public async getParticipant(
+    participantMri: string,
+    options: GetParticipantOptions = {}
+  ): Promise<CallParticipant> {
+    const result = await this.callConnectionImpl.getParticipant(
+      this.callConnectionId,
+      participantMri,
+      options
+    );
+    const callParticipant: CallParticipant = {
+      identifier: result.identifier
+        ? communicationIdentifierConverter(result.identifier)
+        : undefined,
+      isMuted: result.isMuted,
+    };
+    return callParticipant;
+  }
+
+  /**
+   * Get all participants from the call
+   */
+  public async listParticipants(
+    options: GetParticipantOptions = {}
+  ): Promise<ListParticipantsResult> {
+    const result = await this.callConnectionImpl.getParticipants(this.callConnectionId, options);
+    const listParticipantResponse: ListParticipantsResult = {
+      ...result,
+      values: result?.values?.map((acsCallParticipant) =>
+        callParticipantConverter(acsCallParticipant)
+      ),
+    };
+    return listParticipantResponse;
+  }
+
+  /**
+   * Add participants to a call
```

Comment:
Change it to "Add a participant to a call" as now it doesn't support adding a list of participants anymore